viewcontroller/viewcontroller.go

```diff
@@ -108,48 +100,61 @@ func NewRolloutViewController(rolloutId string, noColor bool) *RolloutViewContro
 	}
 }
 
-func (c *RolloutViewController) GetRollout() (rollout.DetailedRollout, error) {
-	detailedRollout := rollout.DetailedRollout{}
+func (c *RolloutViewController) GetRollout() (*rollout.DetailedRollout, error) {
+	detailedRollout := &rollout.DetailedRollout{}
+	errors := make(chan error)
+	wg := &sync.WaitGroup{}
+	wg.Add(3)
```

Comment:
what does 3 means?

Reply:
3 means that runner will be blocked by the wg.Wait() until 3 goroutines will call wg.Done()

Reply:
which go routines?

Reply:
I think I need an explanation regarding this, we will go over it today